spot_utils/utils.py

```diff
@@ -85,6 +87,36 @@ def _callback(event: int, x: int, y: int, flags: int, param: None) -> None:
     return image_click
 
 
+def get_pixel_from_grounded_sam(rgb: NDArray[np.uint8], text_prompt: str) -> Optional[Tuple[int, int]]:
+    """Pick a pixel in rgb that matches text_prompt.
+
+    This function queries a server hosting a GroundedSAM instance (e.g.
+    https://github.com/chsahit/GroundedSAMEndpoint),
+    in order to get an image mask corresponding to the given text. It then computes the center of this mask and returns this pixel. This assumes the object matching text_prompt contains its center.
+    """
+    # Define the endpoint URL
+    url = "http://sahit.csail.mit.edu:7000/grounded_sam"
```

Comment:
could we make this a command-line argument/something the function takes in? I feel like this will be an easy source of bugs/confusion for people running this for the first time otherwise